xml/System/ObsoleteAttribute.xml

```diff
@@ -69,18 +69,14 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <summary>Marks the program elements that are no longer in use. This class cannot be inherited.</summary>
+    <summary>
+            Marks program elements that are no longer in use.
+            </summary>
     <remarks>
-      <format type="text/markdown"><![CDATA[
-<xref:System.ObsoleteAttribute> is applicable to all program elements except assemblies, modules, parameters, and return values. Marking an element as obsolete informs users that the element may be removed in a future version of the product.
-
-The string assigned to the <xref:System.ObsoleteAttribute.Message> property is emitted by the compiler when the attribute target is used in code. Ideally, the string should provide some workaround or programmatic alternative.
-
-Use the <xref:System.ObsoleteAttribute.IsError> property to indicate to the compiler whether using the <xref:System.ObsoleteAttribute> attribute should cause it to emit an error (<xref:System.ObsoleteAttribute.IsError%2A> is `true`) or a warning (<xref:System.ObsoleteAttribute.IsError%2A> is `false`).
-
-For more information about using attributes, see [Attributes](/dotnet/standard/attributes/).
-      ]]></format>
-    </remarks>
+            Source code imported from 
```

Comment:
@huangmin-ms Can you help me figure out where this is coming from? There are no System.Runtime.xml files in the binaries repo under dotnet.

Reply:
@gewarren  
It is coming from Microsoft.Extensions.Diagnostics.ResourceMonitoring of netframework.
![image](https://github.com/user-attachments/assets/5d9b12dd-8483-4ac3-b26f-d22982a864cf)

Reply:
@carlossanlop Any idea why the Microsoft.Extensions.Diagnostics.ResourceMonitoring.dll XML doc file would have documentation for System.ObsoleteAttribute?

Reply:
@huangmin-ms If there is more than one XML doc file for an API, how does mdoc choose which one to use (since docs are shared for all versions/monikers)?

Reply:
@dotnet/area-system-diagnostics @tarekgh do you folks know where ResourceMonitoring lives? I cannot find it. I wanted to see if it is somehow publishing System.ObsoleteAttribute in its ref.

Reply:
@carlossanlop It's coming from .NET Framework code.

Reply:
> @carlossanlop It's coming from .NET Framework code.

Sounds more like a question for @HeathAr or someone in her team.

Reply:
> @huangmin-ms If there is more than one XML doc file for an API, how does mdoc choose which one to use (since docs are shared for all versions/monikers)?

Firstly, sort alphabetically by moniker name. then sort alphabetically by the the name of the xml.

For example: if one API exits in doc1.xml and doc2.xml of **net-9.0** and doc1.xml and doc2.xml of **netframework-4.8.1**.

It will use **doc2.xml** in **netframework-4.8.1**.

Reply:
@carlossanlop I was mistaken. This is coming from the dotnet/extensions repo. They [inject ObsoleteAttribute for down-level frameworks](https://github.com/dotnet/extensions/blob/ef51c637075afde1e28df3d78e9e955141f05328/src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/Microsoft.Extensions.Diagnostics.ResourceMonitoring.csproj#L15) and the .NET Framework XMLs take precedence (which is unfortunate). What to do about this? @evgenyfedorov2 

@huangmin-ms In your image in [this comment](https://github.com/dotnet/dotnet-api-docs/pull/10395#discussion_r1758128787), what folder do you have open? Do I have access to that?

Reply:
> Not sure why an internal sealed class was documenting in the xml, is it expected?

Yes, it's expected that csc.exe includes non-public APIs in the documentation XML file. See https://stackoverflow.com/questions/14786930/why-does-c-sharp-compiler-includes-non-public-members-in-the-xml-documentation.

Reply:
> 
> @huangmin-ms In your image in [this comment](https://github.com/dotnet/dotnet-api-docs/pull/10395#discussion_r1758128787), what folder do you have open? Do I have access to that?

Send you instructions via Teams.